core/src/test/java/org/apache/iceberg/inmemory/TestInMemoryCatalog.java

```diff
@@ -45,16 +40,4 @@ protected InMemoryCatalog catalog() {
   protected boolean requiresNamespaceCreate() {
     return true;
   }
-
-  @Test
-  public void tableCreationWithoutNamespace() {
-    Assumptions.assumeTrue(requiresNamespaceCreate());
-
-    // this should be moved to CatalogTests at some point, but TestNessieCatalog currently fails
```

Comment:
this was fixed for Nessie by https://github.com/apache/iceberg/pull/7283, so we can move the test now to `CatalogTests`